packages/nextjs/src/tools/codegen/test-data/extract-components/regular-imports/package.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
whats the purpose of this file?

Reply:
make lint happy. This is test data, we don't need anything here really.